ui/components/results.jsx

```diff
@@ -1,18 +1,19 @@
-var React = require('react'),
-    Suite = require('./suite.jsx');
+import React from 'react';
+import Suite from './suite.jsx';
 
-var Results = React.createClass({
-  render: function() {
-    var suites;
-
-    suites = this.props.suites.slice(1).map(function(suite, index) {
-      return <Suite suite={suite} key={index}/>;
+class Results extends React.Component {
+  render() {
+    const filter = this.props.filter;
+    let suites = this.props.suites.slice(1).map(function(suite, index) {
```

Comment:
Either use `const` or inline it in the JSX below. You can also use a fat arrow for that anonymous function.